### ISSUE TITLE
Extra backslash being inserted on json download

### ISSUE THREAD

**Issue by alandtse** [lokalise/lokalise-cli-2-go#47]
Recently I noticed the cli started adding an extra backslash in front of `\n`. It's very possible I have a configuration error but I wasn't previously getting the extra backslash to my knowledge and users have just recently reporting the issue. I do not know if it's relevant, but the issue started happening after I changed my `\n` to `__\n` (with a double space preceding `\n` so my strings would work in markdown).

This is the command I'm running via my [ci](https://github.com/custom-components/alexa_media_player/blob/dev/.github/workflows/semantic_release.yml#L16,L32).
```bash
docker run -v /tmp/lokalise:/opt/dest lokalise/lokalise-cli-2 lokalise2 \
          --token "${{ secrets.lokalise_token }}" \
          --project-id 465185555eee18dd537ca6.39714580 file download \
          --format json \
          --unzip-to /opt/dest \
          --original-filenames=false \
          --export-sort a_z \
          --placeholder-format icu \
          --json-unescaped-slashes=true \
          --indentation 2sp \
          --export-empty-as base \
          --add-newline-eof
```

Here is an example of the cli download of the en.json file (see "description" key)
```json
     "step": {
       "action_required": {
         "description": "**{email} - alexa.{url}**  \\nAmazon will send a push notification per the below message. Please completely respond before continuing.  \\n{message}",
         "title": "Alexa Media Player - Action Required"
       },
```

Here is how it appears in lokalise.
```
**[%1$s:email] - alexa.[%1$s:url]**  
Amazon will send a push notification per the below message. Please completely respond before continuing.  
[%1$s:message]
```

If I manually download it using the website (or using the PR to github), I do not get the extra backslashes.
This is the preview file.
```json
    "step": {
      "action_required": {
        "description": "**{email} - alexa.{url}**  \nAmazon will send a push notification per the below message. Please completely respond before continuing.  \n{message}",
```
Lastly, here are my options on the website.
![image](https://user-images.githubusercontent.com/7086117/94329780-e0238400-ff72-11ea-9d6d-8caa87117ee5.png)

Thanks

**Comment by afestein** [comment]
Try this
`--replace-breaks=false`

**Comment by alandtse** [comment]
Ok that was it. I'm confused on why replace-breaks had this impact given the description:
```
Enable to replace line breaks in exported translations with \n (default true). Use --replace-breaks=false to disable. (default true)
```

When it's set to false, it still provides a line break `\n`. It's just not escaping it.

**Comment by christophemenager** [comment]
I agree with you @alandtse, the usage of replace-breaks=false is very counter intuitive, this issue helped me a lot !

**Comment by batman270395** [comment]
Any update on the fix to this issue? For the time being, we are using replace-breaks=false which is counter intuitive, this behaviour should be fixed.

**Comment by xseignard** [comment]
Hello,
We are facing the same issue.
We use react-18next, and when we use the `<Trans />` component, here's how our translations look like:
![Capture d’écran 2022-02-09 à 14 39 06](https://user-images.githubusercontent.com/515603/153212790-b51efc49-095f-4270-a3af-96a835ecd175.png)

Downloading the translations through the `lokalise-cli-2-go` renders the following:
```json
{
  "test": "Hi <0>{{fullName}}<\/0>"
}
```

no matter the value of `replace-breaks` (`true` or `false`)

This is a pretty annoying as it often breaks our workflow and also because there's no need to escape the `/` char

Any idea what to do to circumvent this `\` insertion?

FIY, here's the script we use:
```bash
./lokalise2 --token <OUR_TOKEN> --project-id <OUR_PROJECT_ID> file download \
  --unzip-to "xxx" \
  --bundle-structure "%LANG_ISO%" \
  --original-filenames=true \
  --format json \
  --add-newline-eof \
  --indentation 2sp \
  --export-empty-as skip \
  --export-sort first_added \
  --plural-format i18next \
  --placeholder-format i18n \
  --replace-breaks=true # or false same issue
```

**Comment by BojanKogoj** [comment]
@xseignard your issue is different.

I managed to fix it as @andrewLab-lokalise suggested, and added `replace-breaks`.
